packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -204,15 +190,13 @@ const AccountWizard = props => {
         return props.realAccountSignup(clone);
     };
 
-    const setAccountCurrency = () => props.setAccountCurrency(form_values().currency);
```

Comment:
can we remove `setAccountCurrency` from PropTypes and connect function if we don't use it anymore.